NtupleTools/test/make_ntuples_cfg.py

```diff
@@ -144,508 +148,358 @@
+    maxDeltaR = cms.double(0.1),
+)
+fs_daughter_inputs['taus'] = 'miniAODTauJetInfoEmbedding'
+process.jetInfoEmbedding = cms.Path(
+    process.miniAODElectronJetInfoEmbedding +
+    process.miniAODMuonJetInfoEmbedding +
+    process.miniAODTauJetInfoEmbedding
+)
+process.schedule.append(process.jetInfoEmbedding)
 
-        process.miniElectronsEmbedIp = cms.EDProducer(
-            "MiniAODElectronIpEmbedder",
-            src = cms.InputTag(fs_daughter_inputs['electrons']),
-            vtxSrc = cms.InputTag("offlineSlimmedPrimaryVertices"),
-        )
-        output_commands.append('*_miniElectronsEmbedIp_*_*')
-        fs_daughter_inputs['electrons'] = 'miniElectronsEmbedIp'
+# mvamet
+if options.runMVAMET:
+    process.load("RecoJets.JetProducers.ak4PFJets_cfi")
+    process.ak4PFJets.src = cms.InputTag("packedPFCandidates")
+    process.ak4PFJets.doAreaFastjet = cms.bool(True)
+    
+    from JetMETCorrections.Configuration.DefaultJEC_cff import ak4PFJetsL1FastL2L3
+    
+    process.load("RecoMET.METPUSubtraction.mvaPFMET_cff")
+    process.pfMVAMEt.srcPFCandidates = cms.InputTag("packedPFCandidates")
+    process.pfMVAMEt.srcVertices = cms.InputTag("offlineSlimmedPrimaryVertices")
+    process.pfMVAMEt.inputFileNames.U     = cms.FileInPath('RecoMET/METPUSubtraction/data/gbrmet_7_2_X_MINIAOD_BX25PU20_Mar2015.root')
+    process.pfMVAMEt.inputFileNames.DPhi  = cms.FileInPath('RecoMET/METPUSubtraction/data/gbrphi_7_2_X_MINIAOD_BX25PU20_Mar2015.root')
+    process.pfMVAMEt.inputFileNames.CovU1 = cms.FileInPath('RecoMET/METPUSubtraction/data/gbru1cov_7_2_X_MINIAOD_BX25PU20_Mar2015.root')
+    process.pfMVAMEt.inputFileNames.CovU2 = cms.FileInPath('RecoMET/METPUSubtraction/data/gbru2cov_7_2_X_MINIAOD_BX25PU20_Mar2015.root')
+    if not options.use25ns:
+        process.pfMVAMEt.inputFileNames.U     = cms.FileInPath('RecoMET/METPUSubtraction/data/gbrmet_7_2_X_MINIAOD_BX50PU40_Jan2015.root')
+        process.pfMVAMEt.inputFileNames.DPhi  = cms.FileInPath('RecoMET/METPUSubtraction/data/gbrphi_7_2_X_MINIAOD_BX50PU40_Jan2015.root')
+        process.pfMVAMEt.inputFileNames.CovU1 = cms.FileInPath('RecoMET/METPUSubtraction/data/gbru1cov_7_2_X_MINIAOD_BX50PU40_Jan2015.root')
+        process.pfMVAMEt.inputFileNames.CovU2 = cms.FileInPath('RecoMET/METPUSubtraction/data/gbru2cov_7_2_X_MINIAOD_BX50PU40_Jan2015.root')
+    
+    process.puJetIdForPFMVAMEt.jec =  cms.string('AK4PF')
+    process.puJetIdForPFMVAMEt.vertexes = cms.InputTag("offlineSlimmedPrimaryVertices")
+    process.puJetIdForPFMVAMEt.rho = cms.InputTag("fixedGridRhoFastjetAll")
+    
+    from PhysicsTools.PatAlgos.producersLayer1.metProducer_cfi import patMETs
+    
+    process.miniAODMVAMEt = patMETs.clone(
+        metSource=cms.InputTag("pfMVAMEt"),
+        addMuonCorrections = cms.bool(False),
+        addGenMET = cms.bool(False)
+    )
+    fs_daughter_inputs['mvamet'] = 'miniAODMVAMEt'
+    
+    process.mvaMetSequence = cms.Path(
+        process.ak4PFJets *
```

Comment:
@dntaylor this and the line after are from your part of the PR but I'm not sure I understand them. MiniAOD already includes AK4PFJets and jet PUID. Can you check with the JME people and make sure we need this? It's causing a big slowdown (especially the jet production).